subprojects/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/extensions/types/InstrumentedTypesResourceGenerator.java

```diff
@@ -44,12 +44,12 @@ public GenerationResult generateResourceForRequests(Collection<CallInterceptionR
         return new GenerationResult.CanGenerateResource() {
             @Override
             public String getPackageName() {
-                return "org.gradle.internal.instrumentation";
+                return "";
             }
 
             @Override
             public String getName() {
-                return "instrumented-classes.txt";
+                return "META-INF/instrumentation/instrumented-classes.txt";
```

Comment:
🤔 Since `META-INF` is a generic Java concept, `META-INF/instrumentation` seems like an opportunity for collisions with some other framework. It would probably make sense to prefix this with something Gradle-specific; maybe `META-INF/gradle/instrumentation` or `META-INF/gradle-instrumentation`.

Reply:
Makes sense. I like `META-INF/gradle/instrumentation` will change it

Reply:
Done